default/scripting/techs/ships/Weapons.focs.txt

```diff
@@ -87,6 +170,43 @@ Tech
     researchturns = 8
     prerequisites = "SHP_ROOT_AGGRESSION"
     unlock = Item type = ShipPart name = "SR_WEAPON_2_1"
+    effectsgroups = [
+        EffectsGroup
+            scope = And [
+                Ship
+                OwnedBy empire = Source.Owner
+                DesignHasPart  name = "FT_HANGAR_1"
+            ]
+            accountinglabel = "FT_HANGAR_1"
+            effects = SetMaxSecondaryStat partname = "FT_HANGAR_1" value = Value + 3
+
+        EffectsGroup
+            scope = And [
+                Ship
+                OwnedBy empire = Source.Owner
+                DesignHasPart  name = "FT_HANGAR_2"
+            ]
+            accountinglabel = "FT_HANGAR_2"
+            effects = SetMaxSecondaryStat partname = "FT_HANGAR_2" value = Value + 2
+
+        EffectsGroup
+            scope = And [
+                Ship
+                OwnedBy empire = Source.Owner
+                DesignHasPart  name = "FT_HANGAR_3"
+            ]
+            accountinglabel = "FT_HANGAR_3"
+            effects = SetMaxSecondaryStat partname = "FT_HANGAR_1" value = Value + 1
```

Comment:
Sorry I should have been more clear ;)
`effects = SetMaxSecondaryStat partname = "FT_HANGAR_1" value = Value + 1`
should that not be
`effects = SetMaxSecondaryStat partname = "FT_HANGAR_3" value = Value + 1`